Source/Fuse.Navigation/Router.ScriptClass.uno

```diff
@@ -201,22 +201,19 @@ namespace Fuse.Navigation
 				if (!request.AddArgument(key, obj[key]))
 					return;
 			}
+
 			request.MakeRequest(r);
 		}
-		
+
 		class ScriptRouterRequest : RouterRequest
 		{
-			Context _context;
-
-			public ScriptRouterRequest( Context context ) :
-				base( Flags.FlatRoute )
+			public ScriptRouterRequest() : base(Flags.FlatRoute)
 			{
-				_context = context;
 			}
 			
 			protected override Node ParseNode(object value)
 			{
-				return _context.Wrap(value) as Node;
+				return value as Node;
```

Comment:
This function could just be replaced now with `as Node` in the base class. It's purpose was just to abstract access to the `Context`, which is now gone. Hmm, possibly the entire `ScriptRouterRequest` is redundant now, it just passes an argument to the base constructor.  Maybe it could be cleaned up later, but leaving as is might be confusing (what purpose does it serve?)

Reply:
Right, yeah. I'll remove this.

Reply:
Fixed!